src/cursor/cur_backup.c

```diff
@@ -181,6 +181,8 @@ __backup_log_append(WT_SESSION_IMPL *session, WT_CURSOR_BACKUP *cb, bool active)
 	if (conn->log) {
 		WT_ERR(__wt_log_get_all_files(
 		    session, &logfiles, &logcount, &cb->maxid, active));
+		F_SET(conn->log, WT_LOG_FORCE_NEWFILE);
+		WT_ERR(__wt_log_force_write(session, true, NULL));
```

Comment:
This method takes a retry boolean, returns an integer (error?) and also has a `did_work` out parameter.

The out-paramter is immediately be set to true, but I do believe this method call can return `0`, and also have `did_work` set to false. I'm not sure if I'm using this method in the appropriate way for what needs to be guaranteed.